app/Config/Routes.php

```diff
@@ -60,10 +59,12 @@
 $routes->get('dashboard/team-member/event/(:num)', [TeamMemberDashboard::class, 'get'], ['filter' => TeamMemberAuthFilter::class]);
 $routes->post('dashboard/team-member/event/(:num)', [TeamMemberDashboard::class, 'createOrUpdate'], ['filter' => TeamMemberAuthFilter::class]);
 
-$routes->get('dashboard/user/social-media-link', [UserDashboard::class, 'get'], ['filter' => 'SpeakerOrTeamMemberAuthFilter']);
-$routes->post('dashboard/user/social-media-link', [UserDashboard::class, 'create'], ['filter' => 'SpeakerOrTeamMemberAuthFilter']);
-$routes->put('dashboard/user/social-media-link', [UserDashboard::class, 'update'], ['filter' => 'SpeakerOrTeamMemberAuthFilter']);
-$routes->delete('dashboard/user/social-media-link/(:num)', [UserDashboard::class, 'delete'], ['filter' => 'SpeakerOrTeamMemberAuthFilter']);
+// The social media link logic resides in the abstract base class ContributorDashboard. Since that class cannot be
+// instantiated, we use the TeamMemberDashboard class here instead.
+$routes->get('dashboard/user/social-media-link', [TeamMemberDashboard::class, 'getLatestSocialMediaLinksForCurrentUser'], ['filter' => 'SpeakerOrTeamMemberAuthFilter']);
```

Comment:
Am I get that right, that this is just refactoring and in the frontend will nothing change?

Reply:
Exactly. I only changed were the code resides, but the route itself stays the same :)